holoviews/plotting/bokeh/chart.py

```diff
@@ -203,3 +203,53 @@ def get_data(self, element, ranges=None):
                 err_xs.append((x, x))
                 err_ys.append((y - neg, y + pos))
         return (dict(xs=err_xs, ys=err_ys), self._mapping)
+
+
+class SpikesPlot(PathPlot):
+
+    color_index = param.Integer(default=1, doc="""
+      Index of the dimension from which the color will the drawn""")
+
+    spike_height = param.Number(default=0.1)
```

Comment:
A better name for this would be nice although I don't have any better suggestions right now..